automation_testing/tests/FormFields/formFieldImageVideoLinkDocumentBrowsing.spec.ts

```diff
@@ -105,7 +115,7 @@ test.describe("FormFields - FormFieldTable - Kitchen Sink", () => {
 		await ffImageVideoLinkDocumentBrowsingPage.documentButton.click();
```

Comment:
This is the only error I'm getting when running the tests: 
![image](https://user-images.githubusercontent.com/23409284/191855089-4a35e99c-d3d9-4deb-bc27-506153ae8e35.png)